### PR TITLE
897889- Removed the VS 2008,2010,2012, and 2013 for the WPF platform.

### PR DESCRIPTION
**Pull request by YuvarajMasilamani**
Hi @SubashiniMahendran 

I have removed the VS 2008,2010,2012, and 2013 for the WPF platform. Please review and merge the changes.

Regards 
Yuvaraj M

### CODE REVIEW

File: wpf/Charts/CodedUI.md [hunk 1]
```diff
@@ -54,12 +54,6 @@ Place this assembly in below mentioned location, and install this assembly in GA
 
 The above assembly must be placed into the following directory based on your Visual Studio version.
 
-For Visual Studio 2010: C:\Program Files (x86)\Common Files\Microsoft Shared\VSTT\10.0\UITestExtensionPackages
-
-For Visual Studio 2012: C:\Program Files (x86)\Common Files\Microsoft Shared\VSTT\11.0\UITestExtensionPackages
-
-For Visual Studio 2013: C:\Program Files (x86)\Common Files\Microsoft Shared\VSTT\12.0\UITestExtensionPackages
-
```
Comment (SubashiniMahendran) [inline wpf/Charts/CodedUI.md:62]: check and include VS2015

File: wpf/DataGrid/UI-Automation.md [hunk 2]
```diff
@@ -57,12 +57,6 @@ To test SfDataGrid with CUITs, build the Extension Project and place it in the m
 
 The above assembly must be placed into the following directory based on your Visual Studio version.
 
-For `Visual Studio 2010` : C:\Program Files (x86)\Common Files\Microsoft Shared\VSTT\10.0\UITestExtensionPackages
-
-For `Visual Studio 2012` : C:\Program Files (x86)\Common Files\Microsoft Shared\VSTT\11.0\UITestExtensionPackages
-
-For `Visual Studio 2013` : C:\Program Files (x86)\Common Files\Microsoft Shared\VSTT\12.0\UITestExtensionPackages
-
```
Comment (SubashiniMahendran) [inline wpf/DataGrid/UI-Automation.md:65]: check and include VS2015

File: wpf/Grouping/Overview.md [hunk 3]
```diff
@@ -49,9 +49,6 @@ The prerequisites details are listed below:
 Development Environments</td><td>
 <ul>
 <li> Visual Studio 2015 (Ultimate, Premium, Professional, and Express)</li>
-<li> Visual Studio 2012 (Ultimate, Premium, Professional, and Express)</li>
-<li> Visual Studio 2010 (Ultimate, Premium, Professional, and Express)</li>
-<li> Visual Studio 2008 (Team System, Professional, Standard & Express)</li>
 <li> Visual Studio 2005 (Professional, Standard & Express)</li></ul></td></tr>
 <tr>
```
Comment (SubashiniMahendran) [inline wpf/Grouping/Overview.md:53]: remove VS2005

File: wpf/Sparkline/Getting-started.md [hunk 4]
```diff
@@ -17,8 +17,6 @@ Following steps explain how to create sparkline,
 
 * Open the [Add Reference](https://www.microsoft.com/en-us/download/details.aspx?id=55984) window from your project.
 * To Choose our assemblies follow the below step depending upon the developing environment. 
-* If using VS 2012 choose Assemblies > Extensions > Syncfusion.SfChart.WPF.dll 
-* If using VS 2010 choose .Net>Syncfusion.SfChart.WPF.dll
 * Add the following namespace in your XAML page:
```
Comment (SubashiniMahendran) [inline wpf/Sparkline/Getting-started.md:20]: check with concern team

File: wpf/Range-Selector/Getting-Started.md [hunk 5]
```diff
@@ -29,9 +29,6 @@ The following section explains how to create SfDateTimeRangeNavigator.
 1. Open the Add Reference window in your project.
 2. Choose our assemblies by following the given steps depending upon the developing environment.
 
-   * If you use VS 2012, choose Assemblies > Extensions > Syncfusion.SfChart.WPF.dll.
-   * If you use VS 2010, choose .Net > Syncfusion.SfChart.WPF.dll.
-
```
Comment (SubashiniMahendran) [inline wpf/Range-Selector/Getting-Started.md:34]: check with concern team as discussed

**Review by SubashiniMahendran — CHANGES_REQUESTED** [review]

File: wpf/Grouping/Overview.md (hunk 3 above)
Comment (YuvarajMasilamani) [reply wpf/Grouping/Overview.md:53]: removed

File: wpf/Charts/CodedUI.md (hunk 1 above)
Comment (YuvarajMasilamani) [reply wpf/Charts/CodedUI.md:62]: updated

File: wpf/DataGrid/UI-Automation.md (hunk 2 above)
Comment (YuvarajMasilamani) [reply wpf/DataGrid/UI-Automation.md:65]: resolved

File: wpf/Sparkline/Getting-started.md (hunk 4 above)
Comment (YuvarajMasilamani) [reply wpf/Sparkline/Getting-started.md:20]: i have checked with the concern team they told me to remove the references

File: wpf/Range-Selector/Getting-Started.md (hunk 5 above)
Comment (YuvarajMasilamani) [reply wpf/Range-Selector/Getting-Started.md:34]: changed

**Review by SubashiniMahendran — APPROVED** [review]

**Pull request merged**